docs/application-connector/05-04-connector-service.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Connector Service sub-chart
+type: Configuration
+---
+
+To configure the Connector Service sub-chart, override the default values of its `values.yaml` file. This document describes parameters that you can configure.
+
+>**TIP:** To learn more about how to use overrides in Kyma, see the following documents: 
+>* [Helm overrides for Kyma installation](/root/kyma/#configuration-helm-overrides-for-kyma-installation)
+>* [Sub-charts overrides](/root/kyma/#configuration-helm-overrides-for-kyma-installation-sub-chart-overrides)
+
+## Configurable parameters
+
+This table lists the configurable parameters, their descriptions, and default values:
+
+| Parameter | Description | Default value |
+|-----------|-------------|---------------|
+| **deployment.args.tokenLength**| Specifies a number of characters of a registration token. | `64` |
+| **deployment.args.appTokenExpirationMinutes** | Specifies a period of time after which a token for an Application expires. It is provided in minutes. | `5` |
+| **deployment.args.runtimeTokenExpirationMinutes** | Specifies a period of time after which a token for a runtime expires. It is provided in minutes. | `10` |
+| **deployment.args.appValidityTime** | Specifies a period of time during which certificates that the service issues for an Application are valid. It is provided in days. | `90` |
```

Comment:
The value contains the unit e.g. '90d' for 90 days or '90h' for 90 hours or '90m' for 90 minutes.

Reply:
Updated also for **deployment.args.runtimeValidityTime**.